src/Avalonia.Controls.ColorPicker/Themes/Fluent/ColorView.xaml

```diff
@@ -0,0 +1,643 @@
+                </Border>
+              </TabItem.Header>
+              <ListBox Items="{TemplateBinding PaletteColors}"
+                       SelectedItem="{Binding Color, RelativeSource={RelativeSource TemplatedParent}, Mode=TwoWay}"
+                       UseLayoutRounding="False"
+                       Margin="12">
+                <ListBox.Styles>
+                  <Style Selector="ListBoxItem">
+                    <Setter Property="Background" Value="Transparent" />
+                    <Setter Property="Padding" Value="0" />
+                    <Setter Property="Template">
+                      <ControlTemplate>
+                        <Grid UseLayoutRounding="False">
+                          <ContentPresenter Name="PART_ContentPresenter"
+                                            Background="{TemplateBinding Background}"
+                                            BorderBrush="{TemplateBinding BorderBrush}"
+                                            BorderThickness="{TemplateBinding BorderThickness}"
+                                            CornerRadius="{TemplateBinding CornerRadius}"
+                                            ContentTemplate="{TemplateBinding ContentTemplate}"
+                                            Content="{TemplateBinding Content}"
+                                            Padding="{TemplateBinding Padding}"
+                                            VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
+                                            HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}" />
+                          <Rectangle x:Name="BorderRectangle"
+                                     IsHitTestVisible="False"
+                                     StrokeThickness="3"
+                                     HorizontalAlignment="Stretch"
+                                     VerticalAlignment="Stretch" />
+                        </Grid>
+                      </ControlTemplate>
+                    </Setter>
+
+                    <Style Selector="^ /template/ Rectangle#BorderRectangle">
+                      <Setter Property="Stroke" Value="{DynamicResource SystemControlHighlightListAccentLowBrush}" />
+                      <Setter Property="Opacity" Value="0" />
+                    </Style>
+
+                    <Style Selector="^:pointerover /template/ Rectangle#BorderRectangle">
+                      <Setter Property="Stroke" Value="{DynamicResource SystemControlHighlightListAccentLowBrush}" />
+                      <Setter Property="Opacity" Value="1" />
+                    </Style>
+
+                    <Style Selector="^:selected /template/ Rectangle#BorderRectangle">
+                      <Setter Property="Stroke" Value="{Binding Converter={StaticResource ContrastBrush}, ConverterParameter={DynamicResource TextControlForeground}}" />
```

Comment:
ConverterParameter doesn't accept bindings (including DynamicResource) unfortunately. Inside of the converter it will be just a InstancedBinding object instead of actual resource. Multibinding can be used instead.

Reply:
Unless https://github.com/AvaloniaUI/Avalonia/pull/7624

Reply:
I know bindings don't work until #7624; however, this code always seemed to work. That parameter is used if alpha threshold is below 50%. If you are sure, I will retest.

Reply:
You are of course correct here, the binding itself is being passed as the parameter.

I was thinking of the Slider which does need to do this type of logic (and it is working); however, it's done in a completely different way so has no issue.

For the Palette here its extremely rare for a color (defined in a palette) to actually have any transparency let alone over 50%. So I have just disabled that logic by removing the parameter itself. If ever Avalonia supports this it can be re-enabled. Instructions and comments are added to the code.